uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -22,16 +22,18 @@ define([
   function LeaveBalanceTabController ($q, $scope, AbsencePeriod, AbsenceType,
     LeaveBalanceReport, notification, Session) {
     var filters = {};
-    var pageSize = 50;
     var vm = this;
 
     vm.absencePeriods = [];
     vm.absenceTypes = [];
     vm.loading = { component: true, report: true };
     vm.loggedInContactId = null;
+    vm.pagination = { page: 1, size: 50 };
```

Comment:
Here the size is set to 50 but in the screenshot it is showing 10 at a time. Is the screenshot not updated or there is something wrong with the logic.